### DISCUSSION TITLE
Feature Request : Show notification bubble when app has warnings

### DISCUSSION THREAD

**Discussion by simonc56** [linuxserver/Heimdall#697] [Ideas]
Some apps like Radarr, Sonarr, Medusa, Sickrage,... have an API to fetch warnings and errors notifications.
I see that Radarr and Sonarr apps already have an enhanced display thanks to their API.
Using those API, could we also have small notification bubbles showing warnings and errors of the app ?

Example here, 2 system errors :

![image](https://user-images.githubusercontent.com/10365746/73118966-bafdf200-3f5b-11ea-8ccc-bd51c78b8af3.png)

**Comment by KodeStar** [comment]
Does the API expose that data? If it does I should be able to add something to V3

**Comment by simonc56** [comment]
Which api are you asking about?
If you talk about softwares in my first comment, then the answer is already written.
